Tests don't pass for forks

Currently [the `test_repository_custom_config` test assumes that the repo's parent folder](https://github.com/leblancfg/portray/blob/c00235ab5f9c18f7f098b9ee474305dd03a73130/tests/test_config.py#L85) is called `timothycrosley`. 

```py
In [10]: config.repository(project_dir)
Out[10]:
{'repo_url': 'https://github.com/leblancfg/portray',
 'repo_name': 'portray',
 'edit_uri': 'edit/master/'}
```

```sh
===================================== FAILURES ======================================
___________________________ test_repository_custom_config ___________________________
project_dir = '/Users/leblancfg/src/github.com/leblancfg/portray'

    def test_repository_custom_config(project_dir):
>       assert config.repository(project_dir) == {
            "edit_uri": "edit/master/",
            "repo_name": "portray",
            "repo_url": "https://github.com/timothycrosley/portray",
        }
E       AssertionError: assert {'edit_uri': ...ncfg/portray'} == {'edit_uri': ...sley/portray'}
E         Omitting 2 identical items, use -vv to show
E         Differing items:
E         {'repo_url': 'https://github.com/leblancfg/portray'} != {'repo_url': 'https://github.com/timothycrosley/portray'}
E         Use -v to get the full diff

tests/test_config.py:86: AssertionError
```

As I've forked the repo, my url is now `.../leblancfg/portray`, and the asserts in this test fail.

It would make sense e.g. to either mock `git.Repo`, or not to test the full `'repo_url'` string in the test.

--- a/tests/test_config.py
+++ b/tests/test_config.py
@@ -4,6 +4,8 @@ from hypothesis_auto import auto_test
 
 from portray import config, exceptions
 
+REPO_URL = "https://github.com/timothycrosley/portray"
+
 FAKE_SETUP_FILE = """
 from setuptools import setup
 
@@ -83,22 +85,22 @@ def test_repository_properties():
 
 
 def test_repository_custom_config(project_dir):
-    assert config.repository(project_dir) == {
+    assert config.repository(project_dir, repo_url=REPO_URL) == {
         "edit_uri": "edit/master/",
         "repo_name": "portray",
-        "repo_url": "https://github.com/timothycrosley/portray",
+        "repo_url": REPO_URL,
     }
 
-    assert config.repository(project_dir, repo_name="different_name") == {
+    assert config.repository(project_dir, repo_name="different_name", repo_url=REPO_URL) == {
         "edit_uri": "edit/master/",
         "repo_name": "different_name",
-        "repo_url": "https://github.com/timothycrosley/portray",
+        "repo_url": REPO_URL,
     }
 
-    assert config.repository(project_dir, edit_uri="edit/develop/") == {
+    assert config.repository(project_dir, edit_uri="edit/develop/", repo_url=REPO_URL) == {
         "edit_uri": "edit/develop/",
         "repo_name": "portray",
-        "repo_url": "https://github.com/timothycrosley/portray",
+        "repo_url": REPO_URL,
     }
 
     assert config.repository(

